tests/test_api.rs

```diff
@@ -459,44 +459,44 @@ fn backing_store_segfault() {
   let _setup_guard = setup();
   let array_buffer_allocator = v8::new_default_allocator().make_shared();
   let shared_bs = {
-    assert_eq!(1, v8::SharedRef::use_count(&array_buffer_allocator));
+    array_buffer_allocator.assert_use_count_eq(1);
     let params = v8::Isolate::create_params()
       .array_buffer_allocator(array_buffer_allocator.clone());
-    assert_eq!(2, v8::SharedRef::use_count(&array_buffer_allocator));
+    array_buffer_allocator.assert_use_count_eq(2);
     let isolate = &mut v8::Isolate::new(params);
-    assert_eq!(2, v8::SharedRef::use_count(&array_buffer_allocator));
+    array_buffer_allocator.assert_use_count_eq(2);
```

Comment:
Seems odd this is exposed to the public API.

Reply:
But the alternative is to have the unreliable `use_count()` method in the public API. 
In that light this seems preferable.